docs/windows.md

```diff
@@ -0,0 +1,227 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay
+network to forward packets between nodes. Currently VXLAN and STT tunnels are
+supported.
+
+This page shows how to install antrea-agent on Windows Nodes and register the
+Node to existing kubernetes cluster.
```

Comment:
to **an** existing **Kubernetes** cluster.

Reply:
Done